src/layouts/MainLayout/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MainLayoutStyle = styled.section`
+    width: 60vw;
+    //display: flex;
+    margin: 0 auto;
+    text-align: center;
+`;
+
+export default function MainLayout({ children }) {
```

Comment:
I think this component is not needed anymore. We now know that the common layout code should be placed in `layout.js`. You can use the `text-align` more closely to where the text is